AutomationHelpers/test/WebLibraryTests.cs

```diff
@@ -123,9 +123,7 @@ public void WebLibraryTest_ResponseStatusCode_404()
         {
         	var logger = new TestReportLogger();
         	
-        	Report.AttachLogger(logger);
         	string statusCode = WebLibrary.GetHttpStatusCode("https://httpstat.us/404");
-        	Report.DetachLogger(logger);
         	
         	Assert.AreEqual("404", statusCode, logger.LastLogMessage);

```

Comment:
"logger.LastLogMessage" will be empty/null now, please remove the logger altogether